src/styles/_variables.scss

```
@@ -0,0 +1,11 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  
+  --nav-link-color: #060B35;
+  --blue-color: #00acdc;
+  --text-main-color: #060b35;
```

Why did you create the two variables with the same values?
```suggestion
  --blue-color: #00acdc;
  --text-main-color: #060b35;
```